scripts/liquidation_bot/liquidateUnderwaterBorrowers.ts

```diff
@@ -0,0 +1,60 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import {
+  CometInterface,
+  Liquidator
+} from '../../build/types';
+
+// XXX this is Kovan-specific
+const daiPool = {
+  tokenAddress: '0x6B175474E89094C44Da98b954EedeAC495271d0F',
+  poolFee: 500,
```

Comment:
Why not 100? Seems like that's the most liquid pool: https://info.uniswap.org/#/tokens/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48

Reply:
because here we added support for DAI base asset and basically we want to borrow from one DAI pool with a fee of 500 and then swap to USDC using the second pool with a 100 fee. Still good point. Now thinking about it, I will swap those 2 pools because Comet won't support DAI.

Reply:
Changed to 100 and swap asset pool to 500 instead